loopy/target/cuda.py

```diff
@@ -297,9 +297,36 @@ def cuda_preamble_generator(preamble_info):
 
 # }}}
 
+# {{{ symbol mangler
+
+
+def cuda_symbol_mangler(kernel, name):
```

Comment:
It looks like this is copy-pasted from OpenCL without changes. Why not just import from there and add a comment there that it's also being used in CUDA?

Copy-pasted code is just additional code that someone has to maintain.

Reply:
Done ! Thanks for pointing it out  !